docs/source/usage/security_recommendations.rst

```diff
@@ -112,5 +112,177 @@ Note:
+
+.. prompt:: bash $
+
+  {
+  "oem": {
+    "interface": "idrac-redfish",
+    "data": {
+      "SystemConfiguration": {
+        "Model": "PowerEdge R640",
+        "ServiceTag": "8CY9Z99",
+        "TimeStamp": "Fri Jun 26 08:43:15 2020",
+        "Components": [
+          {
+            [...]
+            "FQDD": "NIC.Slot.1-1-1",
+            "Attributes": [
+              {
+              "Name": "VirtMacAddr",
+              "Value": "00:00:00:00:00:00",
+              "Set On Import": "False",
+              "Comment": "Read and Write"
+              },
+              {
+              "Name": "VirtualizationMode",
+              "Value": "NONE",
+              "Set On Import": "True",
+              "Comment": "Read and Write"
+              },
+            [...]
+            ]
+          }
+        ]
+      }
+    }
+  }
+
+
+**Steps**:
+
+1. Storage options for Configuration settings:
+
+   * Swift Containers (Admins):
+
+     a. Create the containers to be used for configuration mold storage.
+
+     b. For Ironic Swift user that is configured in the [swift] section add read/write access to these containers.
+
+
+   * HTTP servers (Owners):
+
+     a. To use HTTP server with configuration molds, Enable HTTP PUT support.
+
+     b. Create the directory to be used for the configuration mold storage.
+
+     c. Configure read/write access for HTTP Basic access authentication and provide user credentials in [molds]user and [molds]password fields.
+
+
+2. Use the following manual cleaning steps to export configuration to the wanted location. Run the cleaning step during “manageable” state,  replace the cleaning step and args with one of the following options:
+
+   a. **export_configuration** from the existing system with the **export_configuration_location** input parameter to export the configuration to either a web server(Owners) or a Swift container(Admins).
+
+   HTTP Web Server:
+
+   .. prompt:: bash $
+
+     openstack baremetal node clean --clean-steps  '[{"interface": "management","step": "export_configuration","args":{"export_configuration_location": <HTTP-url>}}]' <node>
+
+   Swift Container:
+
+   .. prompt:: bash $
+
+     openstack baremetal node clean --clean-steps  '[{"interface": "management","step": "export_configuration","args":{"export_configuration_location": <swift-container-url>}}]' <node>
+
+3. Restore the selected system configuration mold using the import_configuration manual cleaning step.:
+
+   a. Directly to the original system with minimal or no modification.
+
+   b. Configure a one to many system configuration settings template i.e.. Modify and import the exported system configuration for other systems to replicate. For example, remove sections that do not need to be replicated such as iDRAC connection settings. The configuration mold can be accessed directly from the storage location.
+
+4. Import Settings either from a web server(Owners) or from a Swift containers(Admins) to the required system:
```

Comment:
You can just call this section "Import Settings"

Reply:
Same. No need to specify "Owners" or "Admins".

Reply:
done